en/flight_controller/intel_aero.md

```diff
@@ -67,19 +67,28 @@ After setting up the PX4 development environment, follow these steps update the
 
 ## Connecting a Lidar Lite range finder
 
-The following instructions are for a Lidar Lite V3 connected via I2C. The I2C port on the Aero (labled compass) is used for the external magnetometer (part of the GPS). Therefore a I2C splitter has to be used to connect the Lidar Lite (see picture).
-
-![](../../assets/hardware/Aero_I2C_splitter.JPG)
+The following instructions are for a Lidar Lite V3 connected via I2C. The I2C port on the Aero (labled compass) is used for the external magnetometer (part of the GPS). Therefore, either a I2C splitter has to be used to connect the Lidar Lite (see picture) or the Telemetry port can be used (recommended as it is not being used). In addition it is recommended to use a electrolytic capacitor to reduce spikes in the distance readings (see [here](https://static.garmin.com/pumac/LIDAR_Lite_v3_Operation_Manual_and_Technical_Specifications.pdf) on page 3).
```

Comment:
This is all confusing. 
1. It says "The following instructions are for a Lidar Lite V3 connected via I2C." but then your pinouts cover both using telemetry and i2c ports
2. Is the capacitor used in both cases or only with I2C?
3. I would remove "(Power enable)" and "(Mode Enable) from the table - makes it obvious "not connected". ONly leave if it is useful for someone to know this when making this particular set of connections.
4. In section title use first letter capitalisation "Connecting a Lidar Lite Range Finder"

I can help you reword if you can confirm the above (that this is for both cases).

Reply:
@hamishwillee 
1. Both ports support I2C
2. Yes
3. done
4. done
Could you please check if the new commit is more clear?